Cargo.toml

```diff
@@ -13,13 +13,17 @@ members = [
 # commands like `cargo test` to only run tests from the default package. And
 # `cargo insta test` doesn't have a `--workspace` flag.
 
+[workspace.package]
```

Comment:
FYI you can also share dependency versions in `workspace.dependencies`.  If you were to use it, it seems like your dev-dependencies are the ones that would most likely benefit (like `insta`) but unsure if that is enough to justify it.  Some projects take the approach of having every dependency be in the workspace.  I've not gone to that extreme yet.

Reply:
Thanks. dependabot makes this fairly maintainable, and not everything needs even insta atm, but if that changes we can promote things

Reply:
Yes, dependabot helps and I don't think it has support for `workspace.dependencies`.  As a side tip, I'd recommend checking out Renovate.  Dependabot seems dead in the water for feature development while [Renovate does have workspace.dependencies support](https://github.com/renovatebot/renovate/pull/18831) and its easier to have more complex policies ([clap](https://github.com/clap-rs/clap/blob/master/.github/renovate.json5) vs [typos](https://github.com/crate-ci/typos/blob/master/.github/renovate.json5)).

The one remaining benefit for `workspace.dependencies` for something like `insta` is making it easier to keep the feature flags aligned.  Granted, if you aren't really adding new workspace members, that might not be much of an issue.

Reply:
Great, yes I think renovate also does grouped updates, so we could do that on a schedule rather than manually (we don't want loads of PRs from each patch change).

I probably won't get to this soon — but if anyone sees this and wants to add Renovate, that would be welcome!

Reply:
Depending on your situation, my clap (library, keep build/regular versions within major low) and typos (binary, use latest and greatest) configs are good starting points.